spec/shr_oncology.txt

```diff
@@ -55,24 +55,27 @@ Description:	"Diagnosis of cancer originating in the tissues of the breast, and
 
 // We need to add an example to the IG!
 // Perhaps move timing prefix to component
-// Perhaps make T, N, M separate observations
+// Perhaps make T, N, M separate observations (mlt_20180725: first pass WIP in this file.)
 // Do we want to follow Travis' approach of putting the Cancer Staging Encounter Phase in Encounter?
 Element:		BreastCancerStageTopic
 Based on:		ObservationTopic
 Description:	"The topic class for describing a breast cancer staging observation."
 				TopicCode is MTH#C2216702 "Malignant Neoplasm of Breast Staging"
 0..1			StageTimingPrefix				
-				EvaluationComponent			
+				EvaluationComponent	
+				PanelMembers.Observation
 includes 0..1 	BreastCancerPrimaryTumorClassification
 includes 0..1 	BreastCancerRegionalNodesClassification
 includes 0..1 	BreastCancerDistantMetastasesClassification
-				PanelMembers.Observation
 includes 0..1	HER2ReceptorStatus
 includes 0..1	EstrogenReceptorStatus
 includes 0..1	ProgesteroneReceptorStatus
-includes 0..1	DCISNuclearGrade
-includes 0..1	BreastCancerHistologicGrade	
+includes 0..1	BreastCancerNuclearGrade
+includes 0..1	BreastCancerHistologicGrade
+includes 0..1	Ki-67NuclearAntigen
```

Comment:
Move this out.  Not directly relevant to stage calculations.

Reply:
completed change on local branch.  pending completion of group review to commit.